7_drivers_munge/src/GCM_driver_utils.R

```diff
@@ -1,3 +1,105 @@
+# Pull GCM crs from GCM netcdf
+pull_gcm_crs <- function(gcm_data) {
+  # pull projection information
+  gcm_map <- ncdf4::ncatt_get(gcm_data, 'rcm_map')
+  gcm_crs <- sprintf("+proj=lcc +lat_0=%s + lon_0=%s +lat_1=%s +lat_2=%s +x_0=0 +y_0=0 +ellps=WGS84 +units=m",
+                     gcm_map$latitude_of_projection_origin,
+                     gcm_map$longitude_of_central_meridian,
+                     gcm_map$standard_parallel[1],
+                     gcm_map$standard_parallel[2])
+  return(gcm_crs)
+}
+
+# Reconstruct GCM grid from a GCM netcdf
+reconstruct_gcm_grid <- function(gcm_nc) {
+  gcm_data <- ncdf4::nc_open(gcm_nc)
+
+  # pull projection information
+  gcm_crs <- pull_gcm_crs(gcm_data)
+
+  # set grid size (there is a 'grid_size_in_meters' attribute,
+  # but I can't seem to pull it with ncatt_get)
+  grid_size_m <- 25000
```

Comment:
Some of these GCM pieces are already hard-coded. It seems like it would be easies to just hard-code in some of the grid configurations and skip downloading and extracting the netCDF attributes. Seems like you could get away with a yaml that had these components, was read in as a list target, and then passed in as arguments to `reconstruct_gcm_grid()` and `construct_grid_tiles()`.

```yaml
# Grid configs
gcm_crs: 
grid_size_m: 
x_min: 
y_min: 
n_cell_x: 
n_cell_y: 
```